demos/finserv-and-insurance/simple-loan-marketplace/README.md

```diff
@@ -0,0 +1,308 @@
+1. Edit the `registration.json` file changing
+    1. `name` - keep the `test-` prefix but make the remainder of the name unique
+    1. `userId` - on both nodes should match your Vendia Share `userId` (i.e. your email address)
+1. Create the Uni
+    1. `share uni create --config registration.json`
+
+Wait about 5 minutes for the Uni to reach a `Running` state.
+
+## Step 2 - Explore the Uni
+Using the Share Web App, explore the Uni and its nodes in more detail.
+
+### Uni Views
+Select a Uni by name from the Uni listing to view Uni details.  These views are accessible directly under the Uni's name from the Uni details view.
+
+* __Partner Nodes__ - A list of all partner nodes.  In this demo, there are none as you're acting on behalf of both participants.
+* __Schema__ - The schema for the Uni, which is identical to [schema.json](uni_configuration/schema.json)
+* __Transactions__ - All transactions of the Uni, and their corresponding block in the distributed ledger 
+* __Uni Settings__ - Participant listing and Uni management functions
+
+### Node Configuration Views
+Click the `Manage node` button to view node details.  These views are accessible using the left menu of the Node details view.
+
+* __Resources__ - All underlying cloud resources for the node
+* __Success Notifications__ - Integrations used to notify when a successful transaction occurs in the Uni
+* __Error Notifications__ - Integrations used to notify when a failed transaction occurs on the Node
+* __Ingress__ - Management of inbound, asynchronous processing channels
+
+### Node Views
+* __Entity Explorer__ - A visual tool for exploring and changing the scalar contents of the node
+* __GraphQL Explorer__ - A GraphQL-centric tool for exploring and changing the contents of the node
+* __File Explorer__ - A visual tool for exploring and changing the file contents of the node
+* __Smart Contracts__ -  A tool for managing the Smart Contracts of the node
+
+## Step 3 - Explore the Pre-Loaded Data of the Uni
+The `initial-state.json` file referenced in Step 1 was used to seed the Uni with data.  Creating a Uni based on an initial, known set of data can simplify development and testing.
+
+You can now explore that data using several of the Node Views mentioned in the last step.
+
+### Explore Data using Entity Explorer
+You can view data using the Entity Explorer view from either node in the Uni.
+
+1. Click `Entity Explorer` on either node
+1. View both `Loan` items
+    1. Columns allow sorting
+    1. Clicking on an `_id` value shows a detailed view of the `Loan`
+    1. Clicking on `View history` from the detailed view shows all version of the `Loan` over time
+1. View the `Loan Perforamnce` items
+    1. Repeat the same sub-steps from above
+    
+### Explore Data using GraphQL Explorer
+You can view data using the GraphQL Explorer view from either node in the Uni.
+
+1. Click `GraphQL Explorer` on either node
+1. List all `Loan` items
+    ```
```

Comment:
```suggestion
    ```graphql
```

Reply:
Alright, I'll do it.  My IDE gets weird when I include GQL but don't link to a GQL schema file.